### DISCUSSION TITLE
I think tutorial documentation should be updated.

### DISCUSSION THREAD

**Discussion by jiseongg** [microsoft/vcpkg#36038] [General]
Hi, I have just started learning Vcpkg.

I am just following the tutorial step by step, but there are several errors in documentation, all of which are minor but confusing to newcomer.

For example, in [Tutorial: Create and publish packages - 4 - Set up the port files](https://learn.microsoft.com/en-us/vcpkg/get_started/get-started-packaging?pivots=shell-bash#4---set-up-the-port-files), there are inconsistency in `portfile.cmake` example and its detailed description. (e.g., `REPO Microsoft/vcpkg-docs` vs. `REPO JavierMatosD/vcpkg-sample-library`)

Also, in that page, some commands are still for **CMD** even if I check **Bash** for Terminal options.

I am doing a lot of trial and errors on my own in following the tutorial, and, thus, facing too many errors :(

P.S., Even if I manage to find correct setup, there seems to be errors in running the commands in my environment (with CMake 3.26.4 in MacOS Sonoma 14.2.1, Apple M2), but I am not reporting it in detail since it might be the result of inconsistent documentation.

Thanks!

**Comment by autoantwort** [comment]
> Also, in that page, some commands are still for CMD even if I check Bash for Terminal options.

You mean `vcpkg` instead of `./vcpkg` ?

**Reply by jiseongg** [reply]
Oh, it was about `--overlay-ports=C:\path\to\custom-overlay` in section 5. Sorry that I skipped the details.
